app/views/schools/placement_dates/subject_selections/new.html.erb

```diff
@@ -1,5 +1,7 @@
 <% self.page_title = 'Select school experience subjects' %>
 
+<%= link_to 'Back', :back, class: 'govuk-back-link' %>
```

Comment:
I think this will actually need to be a link back to the previous controller.

If the user submits something invalid to the form - they'll be on the `create` action (rendering `new`) - the previous page in the browser stack will be the `new` action, not the prior page which is what they'd expect?

Reply:
Ah yeah, good catch